files/scripts/healdata/heal-cedar-data-ingest.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import json
+import requests
+import os
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--directory", "-d", help="Cedar Directory ID for registering ")
+parser.add_argument("--access_token", "-a", help="User access token")
+
+args = parser.parse_args()
+
+if not args.directory:
+    print("Directory ID is required!")
+	exit(1)
+if not args.access_token:
+	print("User access token is required!")
+	exit(1)
+
+dir_id = args.directory
+access_token = args.access_token
+
+token_header = {"Authorization": 'access_token ' + access_token}
+
+# Get the metadata from cedar to register
+cedar = requests.get("http://revproxy-service/cedar/get_directory/" + dir_id, headers=token_header)
```

Comment:
does this work? I thought `http://revproxy-service/` can only be reach within a pod or from devterm?

if not, maybe we should pass hostname into this script